src/app/src/main/kotlin/org/imperial/mrc/hint/controllers/CalibrateController.kt

```diff
@@ -41,11 +41,13 @@ class CalibrateController(val apiClient: HintrAPIClient, val calibrateDataServic
         return apiClient.getCalibrateResultMetadata(id)
     }
 
-    @GetMapping("/result/data/{id}")
+    @GetMapping("/result/data/{id}/{indicator}")
```

Comment:
So I did potentially think about doing this, but would this not make it a bit slower? I don't know about the speeds of querying for two indicators values in one go vs querying for one indicator on two different threads because we can simultaneously send requests, plus even if one fails we would still get results for the other indicator but with the multiple query params the whole thing would fail. Those were my thoughts but I'm honestly not sure 🤣🤣 if you think it's better to do it with query params I'm more than happy to do that

Reply:
Just supporting a single indicator is probably fine for now, if that's the only way we're going to be using it, but if we do want to support multiple indicators in future, it shouldn't be using two query string parameters with the same name - we could comma separate instead though?